### ISSUE TITLE
Include enable_event_notification in DA manifest

### ISSUE THREAD

**Issue by in-1911** [terraform-ibm-modules/terraform-ibm-secrets-manager#176]
The variable `enable_event_notification` is not included in the DA inputs.
As a result, the user cannot control the behavior of the component in regard to creating the Event Notifications sources / topics.
In a scenario where this DA component is used in a stack and referencing existing SM and EN instances, this will result in a failure when it will try to provision a duplicate ibm_sm_en_registration.

https://github.com/terraform-ibm-modules/terraform-ibm-secrets-manager/blob/3c32c088379e8ec9a84aaff10868ffe72764dc41/solutions/standard/main.tf#L116

Unfortunately in a stack scenario it's not always feasible to pass `existing_event_notification_instance_crn = null` for just one of the components while using a real reference for others.
The user should be able to pass existing references and have the DA just return them in the output without creating new resources:
```
existing_sm_instance_crn = "crn:...."
existing_event_notification_instance_crn =  "crn:...."
existing_secrets_manager_crn = "crn:..."
skip_kms_iam_authorization_policy = true
skip_event_notification_iam_authorization_policy = true

enable_event_notification = false  # this is not supported in current DA
```

With the parameters above (except for enable_event_notification), the apply step fails with an error:
```
2024/08/06 17:53:24 Terraform apply | module.secrets_manager.ibm_sm_en_registration.sm_en_registration[0]: Creating...
 2024/08/06 17:53:26 Terraform apply | 
 2024/08/06 17:53:26 Terraform apply | Error: CreateNotificationsRegistrationWithContext failed A registration already exists [secrets-manager.00067E]
```

**Comment by in-1911** [comment]
@shemau @toddgiguere - I am sorry I missed that in the PR, but the way the change is implemented in #178 does not address my issues. 
I need to be able to control EN resource provisioning via the enable_event_notification, not by passing the EN CRN.
Because now the logic is pretty much where we started - if EN CRN is not null, enable_event_notification has to be true (i.e. it's effectively ignored).

I need the module to skip creating any EN-related resources or SM settings if `enable_event_notification = false` and ignore the existing EN CRN.

**Comment by shemau** [comment]
Addressed in #184
